### ISSUE TITLE
Can't click on Magia Record, not even with fake touch enabled

### ISSUE THREAD

**Issue by Raxelgrande** [waydroid/waydroid#1071]
**Describe the bug**
First install Magia Record (you need to first install a vpn connected to Japan, tunnelbear is recommended. Then search for "magia record" on the play store. Once it's installed, you can disable tunnelbear).
When you launch the game, at first it accepts inputs, but once you get to this screen it stops accepting them.
![Screenshot_20230828_113941](https://github.com/waydroid/waydroid/assets/93939943/4050647e-0946-4259-b192-95a84ce27334)

**General information (please complete the following information):**
 - Waydroid tools Version: 1.4.1 
 - Waydroid Images Version: Latest GAPPS at the time of writing.
**Desktop (please complete the following information):**
 - OS: Fedora KDE 38
 - GPU: Integrated Vega8 AMD Graphics
 - CPU: AMD Ryzen 5 3500U
 - Kernel version: 6.4.11-200.fc38.x86_64
 - Host mesa version: 23.1.6
 - Desktop: KDE 5.27.7

**Additional context**
I was running with libhoudini installed. 

**Logs (please upload as file)**
I'm going to put the dmseg and logcat in an answer to this issue.
 
[waydroid.log](https://github.com/waydroid/waydroid/files/12456596/waydroid.log)

[waydroid.prop.txt](https://github.com/waydroid/waydroid/files/12456600/waydroid.prop.txt)

**Comment by Raxelgrande** [comment]
[dmseg output.txt](https://github.com/waydroid/waydroid/files/12456916/dmseg.output.txt)
[logcat output.txt](https://github.com/waydroid/waydroid/files/12456917/logcat.output.txt)

**Comment by Quackdoc** [comment]
are you using fake touch globally or with app name? if app name try it globally with '*.*' and see if that works

**Comment by Raxelgrande** [comment]
I was using with app name, tried switching to use it globally, and it didn't work. 
The way to set it up globally is by typing this into a terminal? It's possible I misunderstood how to apply it globally.
waydroid prop set persist.waydroid.fake_touch "*.*"

**Comment by Quackdoc** [comment]
correct, you do need the asterisks, but ill assume github had messed them up, ill see if I can replicate this later

**Comment by Raxelgrande** [comment]
Any update on this? I even tried to run Waydroid under X11 using cage, and it didn't work.

**Comment by marktomm** [comment]
+1

Haven't tried the game op has, but I am experiencing similar issues with lot of web apps.

`waydroid prop set persist.waydroid.fake_touch "*.*"`

My full script:

```shell
sudo waydroid container stop
sudo waydroid session stop

gnome-terminal -- zsh -c 'cage -- waydroid show-full-ui; exec zsh' &
sleep 15

waydroid prop set persist.waydroid.height 750
waydroid prop set persist.waydroid.width 500
waydroid prop set persist.waydroid.fake_touch "*.*"

waydroid app launch org.telegram.messenger

```
